### PR TITLE
build: move dependency

### LINKED ISSUE #15: Dependencies look unrelated to runtime usage
Just wondering if the `dependencies` (like `husky` etc) are used during runtime of this package. These will be force installed when installing this package and affect the dev experience.

### PR DESCRIPTION
**Pull request by Zxilly**
resolve: #15 
Signed-off-by: Zxilly <zhouxinyu1001@gmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@nodece plz review.

File: package.json [hunk 1]
```diff
@@ -42,28 +42,30 @@
   },
   "homepage": "https://github.com/node-casbin/express-authz#readme",
   "dependencies": {
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-prettier": "^3.1.3",
-    "husky": "^4.2.5",
-    "lint-staged": "^10.2.1",
-    "pretty-quick": "^2.0.1"
+    "casbin": "^4.5.0"
```
Comment (nodece) [inline package.json]: We should also move the `casbin` to peerDependencies, and add `casbin` and `express` to devDependencies.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -13,5 +13,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
+      - run: yarn add express
```
Comment (nodece) [inline .github/workflows/main.yml]: unnecessary command.

**Review by nodece — APPROVED** [review]

**Comment by hsluoyz** [comment]
@Zxilly I'm OK with the changes. But I think we should commit yarn.lock (and update .gitingore)

**Comment by Zxilly** [comment]
@hsluoyz have added it

**Pull request merged**